dive_core/command_hierarchy.h

```diff
@@ -280,8 +280,8 @@ class CommandHierarchy
         {
             uint64_t m_addr : 48;
             uint64_t m_opcode : 8;
-            uint64_t m_is_ce_packet : 1;
-            uint64_t m_reserved : 7;
+            uint64_t m_ib_level : 2;
```

Comment:
is 2 bits enough? the enum IbLevel has 5 levels? (kPrimaryRing, kIb1/2/3/4)

Reply:
Oh you might be right. Officially it only goes up to IB3 (or so I thought). But I'm treating cp_set_draw_packets as an additional ib level, which means it can move to IB4.